README.md

```diff
@@ -1 +1,222 @@
+BACKENDLESS_URL="xxxxxxx"
+BACKENDLESS_USERTOKEN="xxxxx"
+BACKENDLESS_TABLE="xxxxx"
+
+BEARER_AUTH_TOKEN="Bearer xxxxx"
+BEARER_AUTH_TOKEN_PROD="Bearer xxxxxx"
+```
+
+and start it `go run app.go`
+
+#### There are several `routes` that can be access
+`/` - GET tree status. returns merkle tree root hash and number of leafs, nodes.
+``` 
+response
+{
+    "status": true,
+    "tree": {
+        "root": "20B4C63E8063464DE5EF6CC60122AC3138F413A53CED28C641BD7867B76F52B9",
+        "length": 301378
+    }
+}
+```
+
+
+`/info/{ethAddress}` - GET info by given ethereum address. Returns index, hash, amount of tokens and migration status of current leaf, and if it is migrated tx hash 
+``` 
+response: 
+{
+    "index": 17,
+    "hash": "5C73535E4E3A596B439554FC5246AF5E9307AB259E6D00D131A1A9422BE352AA",
+    "tokens": "896946881386313856",
+    "migrated": true,
+    "migrateTxHash": "th_2RWiuYoLTVorP8XCdu7yzfJAZ2eThEUwn6XtVy9W7aaU1CijxW"
+}
+```
+
+
+`/hash/{index}` - GET hash by given leaf index. Returns it's hash
+```
+response:
+{
+    "index": 1,
+    "hash": "B4E1FC7C12705A751908BA6438B77342DA68BFC322E346744BA3D38D63636FEE"
+}
+```
+
+
+`/siblings/{index}` - GET intermediary hashes by given leaf index. Returns array of intermediary hashes
+```
+response:
+{
+    "status": true,
+    "hashes": [
+        "D477BC2283558E13BD236EBC12F00BD41825E07F003CFD1C414EAA61323A9B30",
+        "E03C0C82C6073966F29125656A189E477AEBCAB7D3F087492E6ADAA58775031D",
+        "5EF94B5DE05D8CD99B7AC33D94767FC38A80FF9E935100DFA501A038953042E1",
+        "27C8F150F0FF3C98E2E5449FD8CD0DF3F935D4009FE5CBBAECB67C01657A32B4",
+        "A8D5C7ADBD530EC96F5F2286ADE61A88A5394F32EB12A3C5E348158295CA2FCB",
+        "E7C728C723F22F94BBCA7CAD8738E58E11873D87905875AD99C64EA149CCAD39",
+        "A7578FEF2E2993203D438C973FD1BA2FFB7549EB8D9172E643E7ABAE51F3CEA4",
+        "8CA2AB18E655A16D9BE1FE4BAC6C37300362213A21F1868F7C067DC7BF42B4EA",
+        "0E26EA1CBF30F0556C73F9B77C668775E7227F61C446F33FF4C09FB148485A2F",
+        "4140508F328DF0407D8EC70A475C1C771F440A5CD0E1E2AAFF63EA17FC04D284",
+        "7A91D1910EA39A9B3F6F3C361C19FF8AACD8B48E94D8A48096BAEFC70C62F10C",
+        "C9178C00309D658E133C6FD0E26251E8961F454F772DD4DCF9B52CC477D79900",
+        "4BC0DB55DC1B86C79B80D232FD1C8C0A34FE7686761AB05B15F24623B23E52D4",
+        "6B1668C4C38B49042ECED27A1A1A5237B49FA562CD94D4FDB27076DC0602E76E",
+        "A618432138C49B84735552EE60C4DFCE2E7EBF9161C592554F39C800E155E5C1",
+        "C9761373ADDBEEEB73C932B213985FF233C3207D873394D5AF8E8071EA823EA2",
+        "AB0FA2604F5F394B0ADFC2AAE55B8296F390A58910E7CC6E1DC031023941A259",
+        "9FEF092769B79A4ED4540D43386A4EA6A6B8F03A89DB2E3B7F955817EBA05ED4",
+        "16D23EC2BE773B1ABD000772D8FAE6D7502252381E3B21853697FDD0BA0D78B9"
+    ]
+}
+```
+
+
+`/validate` - POST validate is provided data can generate same merkle root hash. Should pass ethereum address, amount of tokens, leaf index and an array of intermediary hashes. Returns is exists
```

Comment:
Returns is exists => 
Returns object with boolean property ‘exists’ which validates if the request is valid or not.